src/coreclr/jit/gentree.h

```diff
@@ -4474,8 +4470,6 @@ struct CallArgABIInformation
     // Unix amd64 will split floating point types and integer types in structs
     // between floating point and general purpose registers. Keep track of that
     // information so we do not need to recompute it later.
```

Comment:
Maybe the comment should be updated too?

Reply:
I left it since the comment equally applies to the `StructDesc` that we do use in a few places.